pkg/service/store/store.go

```diff
@@ -65,6 +65,9 @@ type ClusterService struct {
 	// Backends is map indexed by the backend IP address
 	Backends map[string]PortConfiguration `json:"backends"`
 
+	// Hostnames is map indexed by the backend IP address
+	Hostnames map[string]string `json:"hostnames"`
```

Comment:
I added a new field instead of adding it into Backends because I wasn't sure how to maintain the backward compatibility.

I did this in a separate map instead of including that in the backends one for retro compatibility reasons.

I'm wondering if doing a type like that won't work though:
```
type Backend struct {
	PortConfiguration `json:",inline"`

	Hostname string `json:"hostname"`
}
```
But then as the `PortConfiguration` type is directly a map this would mean that it would be encoded in json within a single dict where some keys are the port name and one of the key is the hostname... This kinda looks weird and I am not even sure it would be decoded correctly...